src/coreclr/tools/aot/ILCompiler.ReadyToRun/Compiler/DependencyAnalysis/ReadyToRun/RuntimeFunctionsTableNode.cs

```diff
@@ -72,31 +71,86 @@ public override ObjectData GetData(NodeFactory factory, bool relocsOnly = false)
             // Add the symbol representing this object node
             runtimeFunctionsBuilder.AddSymbol(this);
 
-            foreach (MethodWithGCInfo method in _methodNodes)
-            {
-                int[] funcletOffsets = method.GCInfoNode.CalculateFuncletOffsets(factory);
+            uint runtimeFunctionIndex = 0;
+            List<uint> mapping = new List<uint>();
 
-                for (int frameIndex = 0; frameIndex < method.FrameInfos.Length; frameIndex++)
+            for (int hot = 0; hot < 2; hot++)
```

Comment:
Apologies for nitpicking but this seems counter-intuitive - if I'm not mistaken, the pass for <code>hot = 0</code> corresponds to the hot code whereas the pass for <code>hot = 1</code> represents the cold code. I would recommend either renaming <code>hot</code> to <code>cold</code> or running the loop backwards from 1 to 0.